pkg/apis/options/options.go

```diff
@@ -126,6 +127,7 @@ func NewFlagSet() *pflag.FlagSet {
 	flagSet.Bool("skip-jwt-bearer-tokens", false, "will skip requests that have verified JWT bearer tokens (default false)")
```

Comment:
As the `access_token` doesn't have to be a jwt token. We should deprecate this flag and add a new one:


```suggestion
	flagSet.Bool("skip-jwt-bearer-tokens", false, "(Deprecated) will skip requests that have verified JWT bearer tokens (default false)")
	flagSet.Bool("skip-bearer-tokens", false, "will skip requests that have verified bearer tokens. Either a jwt based id_token or an access_token (default false)")
```

Reply:
With this functionality, I agree. It made sense before when we only supported JWTs.